tests/unit/NameRequestInfo.spec.ts

```diff
@@ -83,9 +83,7 @@ describe('Name Request Info component', () => {
   })
 
   it('renders the Name Request information with data', async () => {
-    store.state.stateModel.nameRequest = { ...mockNrData }
-
-    await Vue.nextTick()
+    await wrapper.vm.$store.commit('mutateNameRequestState', { ...mockNrData })
```

Comment:
Commit is async so you can await it.

IMO this is cleaner than waiting for a tick.